### DISCUSSION TITLE
Plan to reintroduce fitted models on plots (apart from `PolyFit`)

### DISCUSSION THREAD

**Discussion by Rabeez** [mwaskom/seaborn#3082] [Q&A] [answered]
The functional API (`regplot`) had the option to draw logistic, robust or lowess regressions via statsmodels but the new object API only offers polynomial fit with `PolyFit`.

Is this an intentional choice or will these be added as extensions? 

This gap feels odd since the related `Agg` and `Est` stats have been improved in terms of flexibility.

**Comment by mwaskom** [comment, marked as answer]
Yes, there will be additional stat transforms added in subsequent releases.
